wiki/Beatmap/Beatmap_tags/en.md

```diff
@@ -0,0 +1,207 @@
+# Beatmap tags
+
+*Not to be confused with [TAG difficulties](/wiki/Beatmap/TAG_beatmaps).*
+
+Keywords can be added to beatmaps to make searching for them more convenient. There are two types of tags, each with their own purpose.
+
+## Mapper tags
+
+*For rules regarding mapper tags, see: [Ranking criteria/Metadata § Tags](/wiki/Ranking_criteria/Metadata#tags)*
+
+**Mapper tags** can be used to include useful background information about a beatmap or song, e.g. the album name or the medium for which the song was produced for.
+
+When entering metadata in the [song setup window](/wiki/Client/Beatmap_editor/Song_setup), a beatmap host is able to choose custom keywords by separating them with spaces. Should a keyword itself contain spaces, they must be replaced with an underscore (`_`).
+
+The keywords can be set as a filter in [song select](/wiki/Client/Interface#song-select) as well as the [beatmap listing](https://osu.ppy.sh/beatmapsets) via full-text search.
+
+## User tags
+
+*Note: This feature is not available in osu!(stable).*
```

Comment:
```suggestion
*Note: The ability to add user tags in-game is only avalable in osu!(lazer).*
```

Reply:
I wanted to express here that **neither** searching nor **voting** for user tags is supported in osu!(stable).

This makes the sentence compared to my intention more ambigous in my opinion. This might also conflict with my concern [here](https://github.com/ppy/osu-wiki/pull/13171#discussion_r2051808924).